app/services/users/delete_account_service.rb

```diff
@@ -73,7 +73,9 @@ def create_audit_record
         metadata: {
           email: @user.email,
           account_deletion_notification_sent_at:
-            @user.account_deletion_notification_sent_at&.iso8601
+            @user.account_deletion_notification_sent_at&.iso8601,
+          cohort_ids: @user.founders.map(&:cohort_id).uniq,
+          organisation_id: @user.organisation&.id
```

Comment:
This should be `@user.organisation_id` (also seen in https://github.com/developedbysaad/pupilfirst/commit/e3bf3f0cf4b58eff788ff3d48cf643112e934289), because there's no need to load the orgranisation record just to get its `id`.